Specs/Scene/ExpressionSpec.js

```diff
@@ -2077,6 +2077,21 @@ defineSuite([
 
         expression = new Expression('atan2(1,0)');
         expect(expression.evaluate(frameState, undefined)).toEqualEpsilon(0.5 * Math.PI, CesiumMath.EPSILON10);
+
+        expression = new Expression('atan2(vec2(0,1),vec2(1,0))');
+        expect(expression.evaluate(frameState, undefined).x).toEqualEpsilon(0.0, CesiumMath.EPSILON10);
+        expect(expression.evaluate(frameState, undefined).y).toEqualEpsilon(0.5 * Math.PI, CesiumMath.EPSILON10);
+
+        expression = new Expression('atan2(vec3(0,1,0.5),vec3(1,0,0.5))');
+        expect(expression.evaluate(frameState, undefined).x).toEqualEpsilon(0.0, CesiumMath.EPSILON10);
+        expect(expression.evaluate(frameState, undefined).y).toEqualEpsilon(0.5 * Math.PI, CesiumMath.EPSILON10);
+        expect(expression.evaluate(frameState, undefined).z).toEqualEpsilon(0.25 * Math.PI, CesiumMath.EPSILON10);
+
+        expression = new Expression('atan2(vec4(0,1,0.5,1),vec4(1,0,0.5,0))');
+        expect(expression.evaluate(frameState, undefined).x).toEqualEpsilon(0.0, CesiumMath.EPSILON10);
+        expect(expression.evaluate(frameState, undefined).y).toEqualEpsilon(0.5 * Math.PI, CesiumMath.EPSILON10);
+        expect(expression.evaluate(frameState, undefined).z).toEqualEpsilon(0.25 * Math.PI, CesiumMath.EPSILON10);
+        expect(expression.evaluate(frameState, undefined).w).toEqualEpsilon(0.5 * Math.PI, CesiumMath.EPSILON10);
```

Comment:
For each of these, just evaluate the expression once and do a `toEqualEpsilon` on a Cartesian, like `evaluates pow function` below.